circle/push-artifact-to-s3.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+# Pushes a build artifact for a specific repository to a specified S3 location
+
+info() {
+  echo "usage: $0 [ -l s3_location ] [ -r repository ] [ -a artifact_path ]"
+  echo
+  echo "  -l s3_location        Required: The Amazon S3 location to which to push the artifact"
+  echo "  -r repository         Required: The repository of the project from which the artifact is built"
+  echo "  -a artifact_path      Required: The path to the artifact object (jar, tar, etc.) to store in Amazon S3"
+  echo
+}
+
+parse_options() {
+	while getopts "hl:r:a:e:" opt; do
+		case "$opt" in
+			h)
```

Comment:
The `-h` case will never trigger since the the arg count is checked (line 49) before calling `parse_options`.